<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signup</class>
 <widget class="QDialog" name="signup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>281</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Sign up</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newUsername">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="newPassword">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: white;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createAccountButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CREATE ACCOUNT</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-38</y>
     <width>431</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.517, y2:1, stop:0 rgba(65, 63, 180, 255), stop:1 rgba(170, 0, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>49</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.489, y1:0, x2:0.517, y2:1, stop:0 rgba(170, 0, 255, 255), stop:1 rgba(65, 63, 180, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>groupBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
